src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
+@import "./utils/variables.scss";
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $column-size;
+  gap: 48px;
 
   max-width: 944px;
-  margin: 0 auto;
+
+  @media (min-width: 488px) and (max-width: 767px) {
+      grid-template-columns: repeat(2, $column-size);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+      grid-template-columns: repeat(3, $column-size);
+  }
+
+  @media (min-width: 1024px) {
+      grid-template-columns: repeat(4, $column-size);
+  }
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+
```

Comment:
missed it before, there should be a grid here

Reply:
It's more about main the main container :)